language-design.md

```diff
@@ -16,8 +16,22 @@ are:
 For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
+"We have to do this a lot when we write real computer programs: a thought that seems like a primitive in our minds turns out not to be a primitive in a programming language, and in each new program we must define it once more." [Steele, 1998]
 
+I chose this quote because I thought it was epecially true and interesting. It highlights the differences in various programming languages. Perhaps if you take some primitives to be more important than others then you will perfer languages with those defined primitives. -Daniel
 
+I've certainly found this to be the case in my experience programming. I'd also add that what does or doesn't turn out to be a primitive, as well as what you might expect to be a primitive, depends in some capacity on what you're used to. For a while after taking CS5, for example, I'd tend to want languages like Java to have as primitives bits of functionality that are primitives in Python. One specific example that comes to mind is that in Python, the built-in max (and min) function can be called
+on lists and other collections, whereas Java's Math package only has max and min methods that take two numbers as arguments. -Aaron
+
```

Comment:
I definitely lean toward the second part of what Aaron said. There may be some primates that we just think are more important than others, but I don't necessarily know what languages define them as primitives and I wouldn't go searching for a language that did. More importantly in my mind are our preconceived notions about what should be primitive based on what we have seen in the past. For example, the dictionaries in Python are amazing, they are such a useful tool and should absolutely be primitives, but I often forget that they even exist and I don't tend to think of them as primitives because my first programming language was Java, so I never even thought about needing/wanting/ the usefulness of dictionaries until I started working in Python and even now, I rarely make use of them as much as I could.